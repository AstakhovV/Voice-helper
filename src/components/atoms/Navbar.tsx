import React from 'react';

const Navbar = () => <div>123</div>;

export default Navbar;
