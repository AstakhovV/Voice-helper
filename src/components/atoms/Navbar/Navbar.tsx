import React from 'react';

const Navbar = () => (
  <div className="text-2xl font-bold p-1" role="heading">
    <p>SIBURKA - voice helper</p>
  </div>
);

export default Navbar;
