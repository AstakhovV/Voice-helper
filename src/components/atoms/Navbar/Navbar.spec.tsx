import Navbar from './Navbar';
import React from 'react';
import { render, screen } from '@testing-library/react';

describe('Render Navbar', () => {
  test('Navbar renders correctly', () => {
    render(<Navbar />);
    const navbar = screen.getByRole('heading');
    expect(navbar).toMatchSnapshot();
  });
});
