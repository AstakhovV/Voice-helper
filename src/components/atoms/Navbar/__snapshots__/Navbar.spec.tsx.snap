// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render Navbar Navbar renders correctly 1`] = `
<div
  class="text-2xl font-bold p-1"
  role="heading"
>
  <p>
    SIBURKA - voice helper
  </p>
</div>
`;
